main.py

```diff
@@ -17,11 +17,18 @@
 formatted_response = json.dumps(response_data, indent=4)
 print(formatted_response)
 
+def readCity():
+  while True:
+    city = input("Please enter the name of your city: ")
+    location = city
+    return location
+    break
+
```

Comment:
Also, we need the location returned to be encoded to a url, this link shows a library that does that for us: https://www.urlencoder.io/python/

For example, "New York" needs to be returned as "New%20York"